Phishing protection: unit tests fail

See [this Kokoro failure](https://source.cloud.google.com/results/invocations/a255dd66-093d-44b1-8173-cbe398ef3220/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Fphishingprotection/log), which I can reproduce locally:

```python
_ ERROR collecting tests/unit/gapic/v1beta1/test_phishing_protection_service_client_v1beta1.py _
tests/unit/gapic/v1beta1/test_phishing_protection_service_client_v1beta1.py:21: in <module>
    from google.cloud import phishingprotection_v1beta1
google/cloud/phishingprotection_v1beta1/__init__.py:20: in <module>
    from google.cloud.phishingprotection_v1beta1.gapic import (
google/cloud/phishingprotection_v1beta1/gapic/phishing_protection_service_client.py:40: in <module>
    "google-cloud-phishingprotection"
.nox/unit-3-6/lib/python3.6/site-packages/pkg_resources/__init__.py:481: in get_distribution
    dist = get_provider(dist)
.nox/unit-3-6/lib/python3.6/site-packages/pkg_resources/__init__.py:357: in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
.nox/unit-3-6/lib/python3.6/site-packages/pkg_resources/__init__.py:900: in require
    needed = self.resolve(parse_requirements(requirements))
.nox/unit-3-6/lib/python3.6/site-packages/pkg_resources/__init__.py:786: in resolve
    raise DistributionNotFound(req, requirers)
E   pkg_resources.DistributionNotFound: The 'google-cloud-phishingprotection' distribution was not found and is required by the application
```

--- a/phishingprotection/docs/conf.py
+++ b/phishingprotection/docs/conf.py
@@ -45,6 +45,7 @@ autoclass_content = "both"
 autodoc_default_flags = ["members"]
 autosummary_generate = True
 
+
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
 
@@ -120,6 +121,7 @@ pygments_style = "sphinx"
 
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = True
+
 
 # -- Options for HTML output ----------------------------------------------
 
@@ -229,6 +231,7 @@ htmlhelp_basename = "google-cloud-phishingprotection-doc"
 
 # -- Options for warnings ------------------------------------------------------
 
+
 suppress_warnings = [
     # Temporarily suppress this to avoid "more than one target found for
     # cross-reference" warning, which are intractable for us to avoid while in
@@ -284,6 +287,7 @@ latex_documents = [
 # If false, no module index is generated.
 # latex_domain_indices = True
 
+
 # -- Options for manual page output ---------------------------------------
 
 # One entry per manual page. List of tuples
@@ -300,6 +304,7 @@ man_pages = [
 
 # If true, show URL addresses after external links.
 # man_show_urls = False
+
 
 # -- Options for Texinfo output -------------------------------------------
 
@@ -330,6 +335,7 @@ texinfo_documents = [
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 # texinfo_no_detailmenu = False
 
+
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
     "python": ("http://python.readthedocs.org/en/latest/", None),
@@ -345,6 +351,7 @@ intersphinx_mapping = {
     "fastavro": ("https://fastavro.readthedocs.io/en/stable/", None),
     "pandas": ("https://pandas.pydata.org/pandas-docs/stable/", None),
 }
+
 
 # Napoleon settings
 napoleon_google_docstring = True

--- a/phishingprotection/google/cloud/phishingprotection.py
+++ b/phishingprotection/google/cloud/phishingprotection.py
@@ -14,9 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 from __future__ import absolute_import
 
 from google.cloud.phishingprotection_v1beta1 import PhishingProtectionServiceClient
 from google.cloud.phishingprotection_v1beta1 import types
+
 
 __all__ = ("types", "PhishingProtectionServiceClient")

--- a/phishingprotection/google/cloud/phishingprotection_v1beta1/__init__.py
+++ b/phishingprotection/google/cloud/phishingprotection_v1beta1/__init__.py
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 from __future__ import absolute_import
 
 from google.cloud.phishingprotection_v1beta1 import types

--- a/phishingprotection/google/cloud/phishingprotection_v1beta1/gapic/phishing_protection_service_client.py
+++ b/phishingprotection/google/cloud/phishingprotection_v1beta1/gapic/phishing_protection_service_client.py
@@ -13,6 +13,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 """Accesses the google.cloud.phishingprotection.v1beta1 PhishingProtectionServiceV1Beta1 API."""
 
 import pkg_resources
@@ -36,8 +37,9 @@ from google.cloud.phishingprotection_v1beta1.gapic.transports import (
 from google.cloud.phishingprotection_v1beta1.proto import phishingprotection_pb2
 from google.cloud.phishingprotection_v1beta1.proto import phishingprotection_pb2_grpc
 
+
 _GAPIC_LIBRARY_VERSION = pkg_resources.get_distribution(
-    "google-cloud-phishingprotection"
+    "google-cloud-phishing-protection"
 ).version
 
 

--- a/phishingprotection/google/cloud/phishingprotection_v1beta1/gapic/phishing_protection_service_client_config.py
+++ b/phishingprotection/google/cloud/phishingprotection_v1beta1/gapic/phishing_protection_service_client_config.py
@@ -1,7 +1,10 @@
 config = {
     "interfaces": {
         "google.cloud.phishingprotection.v1beta1.PhishingProtectionServiceV1Beta1": {
-            "retry_codes": {"idempotent": ["UNAVAILABLE"], "non_idempotent": []},
+            "retry_codes": {
+                "idempotent": ["DEADLINE_EXCEEDED", "UNAVAILABLE"],
+                "non_idempotent": [],
+            },
             "retry_params": {
                 "default": {
                     "initial_retry_delay_millis": 100,

--- a/phishingprotection/google/cloud/phishingprotection_v1beta1/gapic/transports/phishing_protection_service_grpc_transport.py
+++ b/phishingprotection/google/cloud/phishingprotection_v1beta1/gapic/transports/phishing_protection_service_grpc_transport.py
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 import google.api_core.grpc_helpers
 
 from google.cloud.phishingprotection_v1beta1.proto import phishingprotection_pb2_grpc

--- a/phishingprotection/google/cloud/phishingprotection_v1beta1/types.py
+++ b/phishingprotection/google/cloud/phishingprotection_v1beta1/types.py
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 from __future__ import absolute_import
 import sys
 
@@ -21,13 +22,14 @@ from google.api_core.protobuf_helpers import get_messages
 
 from google.cloud.phishingprotection_v1beta1.proto import phishingprotection_pb2
 
+
 _shared_modules = []
 
 _local_modules = [phishingprotection_pb2]
 
 names = []
 
-for module in _shared_modules:
+for module in _shared_modules:  # pragma: NO COVER
     for name, message in get_messages(module).items():
         setattr(sys.modules[__name__], name, message)
         names.append(name)
@@ -36,5 +38,6 @@ for module in _local_modules:
         message.__module__ = "google.cloud.phishingprotection_v1beta1.types"
         setattr(sys.modules[__name__], name, message)
         names.append(name)
+
 
 __all__ = tuple(sorted(names))

--- a/phishingprotection/synth.metadata
+++ b/phishingprotection/synth.metadata
@@ -1,19 +1,19 @@
 {
-  "updateTime": "2019-05-08T12:24:28.560729Z",
+  "updateTime": "2019-05-22T17:24:42.783033Z",
   "sources": [
     {
       "generator": {
         "name": "artman",
-        "version": "0.19.0",
-        "dockerImage": "googleapis/artman@sha256:d3df563538225ac6caac45d8ad86499500211d1bcb2536955a6dbda15e1b368e"
+        "version": "0.20.0",
+        "dockerImage": "googleapis/artman@sha256:3246adac900f4bdbd62920e80de2e5877380e44036b3feae13667ec255ebf5ec"
       }
     },
     {
       "git": {
         "name": "googleapis",
         "remote": "https://github.com/googleapis/googleapis.git",
-        "sha": "51145ff7812d2bb44c1219d0b76dac92a8bd94b2",
-        "internalRef": "247143125"
+        "sha": "9fd48dcb59a5fba8464e6dbe6f4c6ca90c7efbaf",
+        "internalRef": "249470705"
       }
     },
     {

--- a/phishingprotection/synth.py
+++ b/phishingprotection/synth.py
@@ -53,6 +53,12 @@ for version in versions:
         "tests/unit/gapic/v1beta1/test_phishing_protection_service_v1_beta1_client_v1beta1.py"
     )
 
+    s.replace(
+        client,
+        "google-cloud-phishingprotection",
+        "google-cloud-phishing-protection",
+    )
+
     files = [client, client_config, transport, unit_test]
     for file_ in files:
         new_name = str(file_).replace("v1_beta1_", "")

--- a/phishingprotection/tests/unit/gapic/v1beta1/test_phishing_protection_service_client_v1beta1.py
+++ b/phishingprotection/tests/unit/gapic/v1beta1/test_phishing_protection_service_client_v1beta1.py
@@ -13,6 +13,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 """Unit tests."""
 
 import mock

